### PR TITLE
Add project task to agenda on create event card

### PR DESCRIPTION
**Pull request by john-botella**
On the Add New Event page, I added the ability to link the event to the project task (or just project if task selection is emlpty)

### CODE REVIEW

File: htdocs/core/class/html.formprojet.class.php [hunk 1]
```diff
@@ -128,10 +127,9 @@ function select_projects($socid=-1, $selected='', $htmlname='projectid', $maxlen
 	 * @param  int     $nooutput           No print output. Return it only.
 	 * @param  int     $forceaddid         Force to add project id in list, event if not qualified
 	 * @param  int     $htmlid             Html id to use instead of htmlname
-	 * @param  string  $morecss            More CSS
 	 * @return int         			       Nb of project if OK, <0 if KO
 	 */
-	function select_projects_list($socid=-1, $selected='', $htmlname='projectid', $maxlength=24, $option_only=0, $show_empty=1, $discard_closed=0, $forcefocus=0, $disabled=0, $mode=0, $filterkey = '', $nooutput=0, $forceaddid=0, $htmlid='', $morecss='maxwidth500')
+	function select_projects_list($socid=-1, $selected='', $htmlname='projectid', $maxlength=24, $option_only=0, $show_empty=1, $discard_closed=0, $forcefocus=0, $disabled=0, $mode=0, $filterkey = '', $nooutput=0, $forceaddid=0, $htmlid='')
```
Comment (eldy) [inline htdocs/core/class/html.formprojet.class.php]: Why removing parameter $morecss ?

Comment (atm-john) [reply htdocs/core/class/html.formprojet.class.php]: not used in function

File: htdocs/core/class/html.formprojet.class.php [hunk 2]
```diff
@@ -101,14 +100,14 @@ function select_projects($socid=-1, $selected='', $htmlname='projectid', $maxlen
 		}
 		else
 		{
-			$out.=$this->select_projects_list($socid, $selected, $htmlname, $maxlength, $option_only, $show_empty, $discard_closed, $forcefocus, $disabled, 0, $filterkey, 1, $forceaddid, $htmlid, $morecss);
-		}
-		if ($discard_closed)
-		{
-			if (class_exists('Form'))
+			$out.=$this->select_projects_list($socid, $selected, $htmlname, $maxlength, $option_only, $show_empty, $discard_closed, $forcefocus, $disabled, 0, $filterkey, 1, $forceaddid, $htmlid);
```
Comment (eldy) [inline htdocs/core/class/html.formprojet.class.php:103]: Why did your remove parameter $morecss ?

File: htdocs/core/class/html.formprojet.class.php [hunk 3]
```diff
@@ -138,7 +137,6 @@ function select_projects($socid=-1, $selected='', $htmlname='projectid', $maxlen
 	 * @param  int     $nooutput           No print output. Return it only.
 	 * @param  int     $forceaddid         Force to add project id in list, event if not qualified
 	 * @param  int     $htmlid             Html id to use instead of htmlname
```
Comment (eldy) [inline htdocs/core/class/html.formprojet.class.php]: Why did you remove parameter $morecss ?

File: htdocs/core/class/html.formprojet.class.php [hunk 4]
```diff
@@ -316,11 +316,9 @@ function select_projects_list($socid=-1, $selected='', $htmlname='projectid', $m
      *  @param	int		$forcefocus		Force focus on field (works with javascript only)
      *  @param	int		$disabled		Disabled
 	 *  @param	string	$morecss        More css added to the select component
-	 *  @param	string	$projectsListId ''=Automatic filter on project allowed. List of id=Filter on project ids.
-	 *  @param	string	$showproject	'all' = Show project info, ''=Hide project info
 	 *	@return int         			Nbr of project if OK, <0 if KO
 	 */
-	function selectTasks($socid=-1, $selected='', $htmlname='taskid', $maxlength=24, $option_only=0, $show_empty='1', $discard_closed=0, $forcefocus=0, $disabled=0, $morecss='maxwidth500', $projectsListId='', $showproject='all')
+	function selectTasks($socid=-1, $selected='', $htmlname='taskid', $maxlength=24, $option_only=0, $show_empty='1', $discard_closed=0, $forcefocus=0, $disabled=0, $morecss='maxwidth500',$projectsListId = false)
```
Comment (eldy) [inline htdocs/core/class/html.formprojet.class.php]: Why removing this parameter showproject ?

File: htdocs/core/class/html.formprojet.class.php (hunk 2 above)
Comment (john-botella) [reply htdocs/core/class/html.formprojet.class.php:103]: hum, it was a test  I do on an another branch. this haven't to be here... I think I have mixed this dev with a test i did during this development. 

So I close this PR and go back to a new one
